python/GafferSceneUI/LightEditor.py

```diff
@@ -354,6 +382,120 @@ def __editSelectedCells( self, pathListing ) :
+					fullAttributes = self.__settingsNode["in"].fullAttributes( path[:-1] )
+					parentValueData = fullAttributes.get( inspector.name(), None )
+					parentValue = parentValueData.value if parentValueData is not None else False
+
+					currentValues.append( currentValue if currentValue is not None else parentValue )
+				else :
+					currentValues.append( currentValue )
+
+		# Now set the value for all plugs, defaulting to `True` if they are not
+		# currently all the same.
+		newValue = not sole( currentValues )
+
+		with Gaffer.UndoScope( self.scriptNode() ) :
+			for inspector, pathInspections in inspectors.items() :
+				for path, inspection in pathInspections.items() :
+					plug = inspection.acquireEdit()
+
+					if isinstance( plug, ( Gaffer.TweakPlug, Gaffer.NameValuePlug ) ) :
+						plug["value"].setValue( newValue )
+						plug["enabled"].setValue( True )
+						if isinstance( plug, Gaffer.TweakPlug ) :
+							plug["mode"].setValue( Gaffer.TweakPlug.Mode.Create )
+					else :
+						plug.setValue( newValue )
+
+		return True
+
+	def __activeInspectionTweaks( self, pathListing ) :
+
+		tweaks = []
+
+		with Gaffer.Context( self.getContext() ) as context :
+			for columnSelection, column in zip( pathListing.getSelection(), pathListing.getColumns() ) :
+				if not isinstance( column, _GafferSceneUI._LightEditorInspectorColumn ) :
+					continue
+				for path in columnSelection.paths() :
+					context["scene:path"] = GafferScene.ScenePlug.stringToPath( path )
+					inspection = column.inspector().inspect()
+					if inspection is not None and inspection.editable() :
+						source = inspection.source()
+						editScope = self.__settingsNode["editScope"].getInput()
+						if (
+							isinstance( source, ( Gaffer.TweakPlug, Gaffer.NameValuePlug ) ) and
+							( editScope is None or editScope.node().isAncestorOf( source ) ) and
+							source["enabled"].getValue()
+						) :
+							tweaks.append( source )
+
+		return tweaks
+
+	def __removableAttributeInspections( self, pathListing ) :
+
+		inspections = []
+
+		with Gaffer.Context( self.getContext() ) as context :
+			for columnSelection, column in zip( pathListing.getSelection(), pathListing.getColumns() ) :
+				if (
+					not isinstance( column, _GafferSceneUI._LightEditorInspectorColumn ) or
+					not isinstance( column.inspector(), GafferSceneUI.Private.AttributeInspector )
+				) :
+					continue
+				for path in columnSelection.paths() :
+					context["scene:path"] = GafferScene.ScenePlug.stringToPath( path )
+					inspection = column.inspector().inspect()
+					if inspection is not None and inspection.editable() :
+						source = inspection.source()
+						editScope = self.__settingsNode["editScope"].getInput()
+						if (
+							( isinstance( source, Gaffer.TweakPlug ) and source["mode"].getValue() != Gaffer.TweakPlug.Mode.Remove ) or
+							editScope is not None
+						) :
+							inspections.append( inspection )
```

Comment:
For the above to make sense, we'd need an `else : return []` here too.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/4948/commits/8e2b59b2402e01a4ee35db5e58913b67bf87ceb9